WireUI/Sources/WireIndividualToTeamMigrationUI/IndividualToTeamMigrationViewController.swift

```diff
@@ -0,0 +1,266 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+import WireDesign
+import WireDomainAPI
+import WireFoundation
+
+public class IndividualToTeamMigrationViewController: UIViewController {
```

Comment:
Just wondering, what is the reason for this to be `UIKit` rather than `SwiftUI`?

Reply:
@netbe preferred the navigation to be done in UIKit.

Reply:
Oh interesting. Perhaps we should revisit this choice some time (outside the scope of this PR). My assumption is the `SwiftUI` version will be simpler but maybe I missing some context.